build.gradle

```diff
@@ -64,63 +63,63 @@ subprojects {
     }
 
     dependencies {
-        common (
+        common(
                 "javax.xml.bind:jaxb-api:2.3.0",
                 "javax.activation:activation:1.1.1",
                 "io.lettuce:lettuce-core:5.1.4.RELEASE",
         )
 
-        dropwizard (
+        dropwizard(
                 "io.dropwizard:dropwizard-core:${dropwizard_version}",
                 "io.dropwizard:dropwizard-client:${dropwizard_version}",
                 "io.dropwizard:dropwizard-views-mustache:${dropwizard_version}",
                 "io.dropwizard:dropwizard-views-mustache:${dropwizard_version}",
                 "uk.gov.ida:dropwizard-logstash:1.3.5-68"
         )
 
-        opensaml (
+        opensaml(
                 "org.opensaml:opensaml-core:${opensaml_version}",
                 "org.opensaml:opensaml-saml-impl:${opensaml_version}",
                 "org.opensaml:opensaml-storage-impl:${opensaml_version}",
         )
 
-        eidas_saml (
+        eidas_saml(
                 "se.litsec.eidas:eidas-opensaml3:1.3.1",
                 "se.litsec.opensaml:opensaml3-ext:1.2.2",
         )
 
-        ida_utils (
+        ida_utils(
                 "uk.gov.ida:rest-utils:${utils_version}"
         )
 
-        verify_saml (
+        verify_saml(
                 "uk.gov.ida:saml-extensions:${saml_libs_version}",
                 "uk.gov.ida:saml-metadata-bindings:${saml_libs_version}",
                 "uk.gov.ida:saml-security:${saml_libs_version}",
                 "uk.gov.ida:saml-utils:${saml_libs_version}",
                 "uk.gov.ida:security-utils:${utils_version}",
         )
 
-        verify_saml_test (
+        verify_saml_test(
                 "uk.gov.ida:common-test-utils:${utils_version}",
                 "uk.gov.ida:saml-metadata-bindings-test:${saml_libs_version}",
         )
 
-        soft_hsm (
+        soft_hsm(
```

Comment:
We can delete the soft_hsm now.

Reply:
yes, got a separate PR for that coming :)